src/convenience/session.ts

```diff
@@ -407,7 +425,7 @@ function fillDefaults<S>(opts: SessionOptions<S> = {}) {
         debug(
             "Storing session data in memory, all data will be lost when the bot restarts.",
         );
-        storage = new MemorySessionStorage();
+        storage = new MemorySessionStorage<S>();
```

Comment:
Is this no longer inferred?

Reply:
this is what I get on the `storage` variable if I remove the annotation:
![image](https://user-images.githubusercontent.com/3948961/219716920-ec4c8f5f-50c0-4197-9632-0e40de53d251.png)

Reply:
Fascinating, I wonder why … but this change is obviously good